motr/io_req.c

```diff
@@ -1445,10 +1445,13 @@ static bool is_node_marked(struct m0_op_io *ioo,
 }
 
 /**
- * Returns number of failed devices or -EIO if number of failed devices exceeds
- * the value of K (number of spare devices in parity group). Once MOTR-899 lands
- * into dev the code for this function will change. In that case it will only
- * check if a given pool is dud.
+ * If everything is fine, returns 0. Otherwise, returns -EIO if
+ * number of failed devices exceeds the value of K (number of parity
+ * units in the parity group). Or a positive number if there are some
+ * failures (devices, processes, nodes) but they are tolerable.
+ *
+ * Once MOTR-899 lands into dev the code for this function will change.
+ * In that case it will only check if a given pool is dud.
```

Comment:
I think this can be removed.

Reply:
We discussed it with @nikitadanilov and agreed that he will look into this jira ticket...